Pollination-apps/custom-chart.md

```diff
@@ -0,0 +1,53 @@
+# Add a custom chart to your Pollination app
+
```

Comment:
An introduction of what are you trying to do in this app will be really helpful.